Misc/NEWS.d/next/Tests/2020-04-20-00-45-56.bpo-40331.oLPd4g.rst

```diff
@@ -0,0 +1 @@
+Increase the coverage for the :mod:`statistics` module by 1%.
```

Comment:
In this case, the news entry should include the tests added, not x% coverage. Otherwise, it's not particularly useful to readers.

Alternatively, I think the news entry could be reasonably skipped for this PR.

Reply:
I agree, just adding a few tests shouldn't be mentioned in NEWS.

Reply:
Thank you both @aeros and @taleinat for your input. Checking similar past PRs (e.g., https://github.com/python/cpython/pull/17949), it seems that `skip news` label is the way to go in this case. I will remove it in the next commit.